src/main/scala/wdl4s/wdl.scala

```diff
@@ -0,0 +1,20 @@
+package wdl4s
+
+import wdl4s.wdl.exception.OutputVariableLookupException
+import wdl4s.wdl.values.WdlValue
+
+import scala.util.{Failure, Try}
+
```

Comment:
Some of these look like they'd be used outside of `wdl`, is the idea that they'd move elsewhere?